### PR TITLE
Add Visualization tool

### PR DESCRIPTION
**Pull request by Hailong-am**
### Description
Add Visualization tool to support find visualizations by its title, and return first 3 result.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VisualizationsTool.java [hunk 1]
```diff
@@ -0,0 +1,165 @@
+package org.opensearch.ml.engine.tools;
```
Comment (ylwu-amzn) [inline ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VisualizationsTool.java:1]: Add license header

File: ml-algorithms/src/test/java/org/opensearch/ml/engine/tools/VisualizationToolTests.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
+package org.opensearch.ml.engine.tools;
```
Comment (ylwu-amzn) [inline ml-algorithms/src/test/java/org/opensearch/ml/engine/tools/VisualizationToolTests.java:1]: add license header

File: ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VisualizationsTool.java (hunk 1 above)
Comment (Hailong-am) [reply ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/VisualizationsTool.java:6]: good catch, add missing header.

File: ml-algorithms/src/test/java/org/opensearch/ml/engine/tools/VisualizationToolTests.java (hunk 2 above)
Comment (Hailong-am) [reply ml-algorithms/src/test/java/org/opensearch/ml/engine/tools/VisualizationToolTests.java:6]: good catch, add missing header.

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**